openwisp_users/migrations/0008_update_admins_permissions.py

```diff
@@ -0,0 +1,15 @@
+from django.db import migrations
+from openwisp_users.migrations import update_admins_permissions
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_users', '0007_unique_email'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            update_admins_permissions, reverse_code=migrations.RunPython.noop
```

Comment:
Oh, there is a manual update. I didn't see it.

Reply:
@NoumbissiValere  I think we should document this like done here: https://github.com/openwisp/openwisp-users#9-create-database-migrations
Should I do it or would you be interested to complete it? :smile:

Reply:
@atb00ker I thought ./manage.py migrate should have done the trick so need for docs

Reply:
@NoumbissiValere  No manual migrations don't get automatically created. (P.S when extending apps, we have to manually create such migration changes as done for `0002`  migration as well, it's manually created in extended apps! :smile: )

Reply:
Oops. Thank you @atb00ker :+1:  didn't know this. Right I am busy with something, when I am done, I will provide the PR. But if you need it urgently, you can go ahead